aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -227,7 +227,9 @@ pub enum ScriptFunctionCall {
     },
 
     /// Withdraw from `account`'s inactive stake.
-    StakeWithdraw {},
+    StakeWithdraw {
```

Comment:
Is this a linter fix? 🤯

Reply:
Crazily enough yes, this came from running, just double checked:
```
cargo clippy --workspace --all-targets
```
Let me know if you think we shouldn't land this because clippy is doing too much.

Reply:
There was a strong desire to add this autogenerated file to the repository. It can be out of date. We could add a check to see if it is up to date. In the meantime, anyone that builds outside of framework may cause this to update.